pkg/get/get.go

```diff
@@ -284,3 +289,73 @@ func GetBinaryName(tool *Tool, os, arch, version string) (string, error) {
 
 	return "", errors.New("BinaryTemplate is not set")
 }
+
+// DownloadList verifies tool exists and returns a download list
```

Comment:
The comment should be updated, `download list` may be `a list of URLs for each tool, for download`